src/main.cpp

```diff
@@ -8538,6 +8538,83 @@ int main_test(int argc, char** argv){
     return vg::unittest::run_unit_tests(argc, argv);
 }
 
+void help_sort(char** argv){
+    cerr << "usage: " << argv[0] << " sort [options] -i input_file -r reference_name > sorted.vg " << endl
+         << "options: " << endl
+         << "           -g, --gfa           input in GFA format" << endl
+         << "           -i, --in            input file" << endl
+         << "           -r, --ref            reference name" << endl
+         << endl;
+}
+
+int main_sort(int argc, char *argv[]) {
+
+    //default input format is vg
+    bool gfa_input = false;
+    string file_name = "";
+    string reference_name = "";
+    
+    int c;
+    while (true) {
+        static struct option long_options[] =
+            {
+                {"gfa", no_argument, 0, 'g'},
+                {"in", required_argument, 0, 'i'},
+                {"ref", required_argument, 0, 'r'},
+                {0, 0, 0, 0}
+            };
+
+        int option_index = 0;
+        c = getopt_long (argc, argv, "i:r:g",
+                         long_options, &option_index);
+
+        /* Detect the end of the options. */
+        if (c == -1)
+            break;
+
+        switch (c)
+        {
+        case 'g':
+            gfa_input = true;
+            break;
+        case 'r':
+            reference_name = optarg;
+            break;
+        case 'i':
+            file_name = optarg;
+            break;
+        case 'h':
+        case '?':
+            /* getopt_long already printed an error message. */
+            help_sort(argv);
+            exit(1);
```

Comment:
All the vg commands return 1 when asked for help. It looks like most other tools do return 0, though.